source/includes/_endpoint-dataset.md

```diff
@@ -486,6 +486,16 @@ in the API.
 
 Categorical variables will be exported with their name instead of their value.
 
```

Comment:
And on second thought, maybe this should be changed to a POST rather than a GET. Looks like we could be potentially constructing large URLs, like if the user supplies a long list of variable ids/URLs to include.